src/server/game/Entities/Player/Player.h

```diff
@@ -1520,6 +1521,7 @@ class TC_GAME_API Player : public Unit, public GridObject<Player>
         void ItemAddedQuestCheck(uint32 entry, uint32 count);
         void ItemRemovedQuestCheck(uint32 entry, uint32 count);
         void KilledMonsterCredit(uint32 entry, ObjectGuid guid, uint32 questId = 0);
+        void KilledMonsterCredit(uint32 entry) { KilledMonsterCredit(entry, this->GetGUID(), 0); };
```

Comment:
Just add a default argument to existing one then. It has the same effect.

Reply:
You cannot use the _this pointer_ in a default argument.

Reply:
I changed this one to match the TC function